Libraries/Network/RCTDataManager.m

```diff
@@ -57,9 +57,11 @@ - (void)executeQuery:(NSString *)queryType
         } else {
           encoding = NSUTF8StringEncoding;
         }
-        int responseCode = (int)[((NSHTTPURLResponse *)response) statusCode];
+        NSHTTPURLResponse *httpResponse = (NSHTTPURLResponse*)response;
+        NSDictionary *headerDict = [httpResponse allHeaderFields];
+        int responseCode = (int)[httpResponse statusCode];
```

Comment:
This should be NSInteger, which is what the method actually returns . Not sure why we'd cast to an int (I appreciate that this was pre-existing, and not something you changed).